test/test_jit.py

```diff
@@ -1583,6 +1583,93 @@ def foo(x):
         test_x = torch.rand(6, 3)
         self.assertEqual(foo(test_x), traced(test_x))
 
+    def test_export_dropout(self):
+        class DropoutTest(torch.nn.Module):
+            def __init__(self):
+                super(DropoutTest, self).__init__()
+                self.dropout = torch.nn.Dropout()
+
+            def forward(self, x):
+                return self.dropout(x)
+
+        test = DropoutTest()
```

Comment:
I'm pretty sure this module is 100% equivalent to just using `torch.nn.Dropout` directly.